pgpubsub/models.py

```diff
@@ -15,13 +15,20 @@
 class Notification(models.Model):
     channel = models.CharField(max_length=MAX_POSTGRES_CHANNEL_LENGTH)
     payload = JSONField()
+    # The field is made nullable to make sure the addiion of the new field is backward
+    # compatible. From the version this change is release the field is effectively non
+    # nullable as in it always gets a value.
+    # After some time the field should be made non nullable here.
+    created_at = models.DateTimeField(null=True)
```

Comment:
Initially I was going to say we should add `auto_now_add=True`, but of course populating this is taken care of by the trigger below. I wonder if it would ever be advantageous to have it anyway - for example if a `Notification` is ever created manually for some reason. I can't think of a concrete use case right now, so maybe we can add that when we need it.

Reply:
I agree

Reply:
@romank0 Yeah totally. Adding support for db_defaults (probably as a third party library) is something I've wanted to work on for a while. I may eventually try to add it as an Opus10 library.

Reply:
@PaulGilmartin how do you feel about this approach https://github.com/Opus10/django-pgpubsub/compare/master...romank0:django-pgpubsub:persistent-notifications-creation-timestamp ?